elements/pfe-tabs/src/pfe-tab.scss

```
@@ -3,17 +3,21 @@
 $LOCAL: tabs;
 
 $LOCAL-VARIABLES: (
-  BackgroundColor: transparent,
-  Color:           pfe-var(ui-disabled--text),
-  link:            pfe-var(link),
-  focus:           pfe-var(link--focus),
-  highlight:       pfe-var(ui-accent),
+  BackgroundColor:       transparent,
+  Color:                 pfe-var(text--muted),
+  link:                  pfe-var(link),
+  text-hover:            #151515,
```

```suggestion
  text--hover:            #151515,
```

Modifiers are separated by double dashes in BEM